package.json

```diff
@@ -31,6 +32,7 @@
     "mysql2": "^2.3.3",
     "nodemon": "^2.0.15",
     "sequelize": "^6.16.2",
-    "tedious": "^14.3.0"
+    "tedious": "^14.3.0",
```

Comment:
well, this looks tedious

Reply:
This was a result of me having to mess with different packages to get H-SJS working. I think there was a point where I'd uninstalled then reinstalled Tedious